content/projects/oop/animals/part2/_index.md

```diff
@@ -156,6 +170,25 @@ Test -> Does cat eat food should Fail
 
 ```
 
+
+### For Python
+
+Use pytest to test your code.
+
+Please refer to the following to find out more: {{% contentlink path="topics/python-specific/automated-testing-with-pytest" %}}
+
+```
+# Python
+
+dog = Dog()
+dog.eats()    # returns 'Food'
+dog.sounds()  # returns 'Bark'
+
+```
+
+Remember the correct naming convention for the tests in your `test_animals.py` file.
```

Comment:
I think this reminder can be made general and put under the general instructions for the reviewer, I've noticed that most learners tend to also look at these instructions when doing their projects. Otherwise, we'll have to remind them under each stream.